### DISCUSSION TITLE
MSSQL database support for superset's backend database.

### DISCUSSION THREAD

**Discussion by praveer38** [apache/superset#30638] [Ideas]
Feature request for superset to support MSSQL database to run as the backend database. As of now, even after configuring an MSSQL database as the backend DB, the migrations are failing.

**Comment by sfirke** [comment]
We note in the FAQ that [MSSQL is specifically _not_ supported as a backend database.](https://superset.apache.org/docs/faq/#what-database-engine-can-i-use-as-a-backend-for-superset)  It is useful to have this confirmed again here, thank you, though sorry you have lost time to this.

It would be a lot of additional work to support MSSQL as a backend DB and I don't believe any current contributors are interested in taking that on.  I feel confident that the only supported backend DBs will continue to be PostgreSQL and MySQL (and SQLite for development but not recommended for production).

In case it helps anyone, I was in a similar situation, at a company that uses Microsoft SQL Server for all application databases.  We set up our first Postgres DB for this, managed by a cloud provider, and it has worked out great.
